test/cmdlineTests/exp_base_literal/output

```diff
@@ -274,8 +274,16 @@ object "C_80" {
 
             }
 
-            function panic_error() {
-                invalid()
+            function panic_error_0x11() {
```

Comment:
Do we want to hardcore the values here?

Reply:
What do you mean by hardcoding? Instead of providing constant arguments to functions? Won't your optimizer step do that anyway with the disadvantage that the function name is less readable? :)

Reply:
I would prefer calling `string YulUtilFunctions::panicFunction(util::PanicCode _code)` with the right code. Hopefully the inliner can inline it, if not the upcoming constant argument specializer should do it.

Also if we are keeping it like this, I would prefer the values in hex.